kfp/doc/setup.md

```diff
@@ -78,7 +78,7 @@ Now, you can create a Kind cluster with all required software installed using th
 ```shell
  make setup
 ```
-from this main package directory or from the `kind` directory.
+from this main package directory or from the `scripts/k8s-setup` directory.
```

Comment:
it should be executed from the `scripts/k8s-setup` directory, and not from the root dir.
The comment is relevant to the other commands below.

Reply:
Fixed. Thanks